core-services/release-controller/_repos/ocp-4.17-rhel94.repo

```diff
@@ -36,7 +36,7 @@ failovermethod = priority
 
 [rhel-9.4-nfv]
 name = rhel-9.4-nfv
-baseurl = https://cdn.redhat.com/content/beta/rhel9/9/x86_64/nfv/os
+baseurl = https://cdn.redhat.com/content/e4s/rhel9/9.4/x86_64/nfv/os/
```

Comment:
minor change: shouldn't this be `eus` too ?

Reply:
https://github.com/openshift/release/pull/51750#discussion_r1592171330